src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/weapons/SwordOfBeheading.java

```diff
@@ -56,27 +59,32 @@ public EntityKillHandler getItemHandler() {
             Random random = ThreadLocalRandom.current();
 
             switch (e.getEntityType()) {
-                case ZOMBIE:
+                case ZOMBIE -> {
                     if (random.nextInt(100) < chanceZombie.getValue()) {
                         e.getDrops().add(new ItemStack(Material.ZOMBIE_HEAD));
                     }
-                    break;
-                case SKELETON:
+                }
+                case SKELETON -> {
                     if (random.nextInt(100) < chanceSkeleton.getValue()) {
                         e.getDrops().add(new ItemStack(Material.SKELETON_SKULL));
                     }
-                    break;
-                case CREEPER:
+                }
+                case CREEPER -> {
                     if (random.nextInt(100) < chanceCreeper.getValue()) {
                         e.getDrops().add(new ItemStack(Material.CREEPER_HEAD));
                     }
-                    break;
-                case WITHER_SKELETON:
+                }
+                case WITHER_SKELETON -> {
                     if (random.nextInt(100) < chanceWitherSkeleton.getValue()) {
                         e.getDrops().add(new ItemStack(Material.WITHER_SKELETON_SKULL));
                     }
-                    break;
-                case PLAYER:
+                }
+                case PIGLIN -> {
+                    if (random.nextInt(100) < chancePiglin.getValue()) {
+                        e.getDrops().add(new ItemStack(Material.PIGLIN_HEAD));
```

Comment:
Do we need a version check here, looks like this skull was added in 1.20? (ignoring the experimental pack one)

Reply:
I was wondering about that but in the checklist they said they tested it across versions so I assumed it failed silently, good question tho